queries.sql

```diff
@@ -188,4 +188,152 @@ FROM animals A
 JOIN owners O
 ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY animal DESC;
\ No newline at end of file
+ORDER BY animal DESC;
+
+/*MANY-TO-MANY QUERIES*/
+
+/*"Who was the last animal seen by William Tatcher?"*/
+
+SELECT TOP 1
+A.name AS animal_name,
+vets.name AS vet_name,
+visits.date_of_visit
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC;
```

Comment:
- [ ] As the previous reviewer mentioned, the query should show only one entry i.e. the last animal that was visited but your screenshot has 3 animals rendered. Kindly check your query and update the screenshot.

![image](https://github.com/Movgto/vet-clinic-database/assets/99715640/f203718d-f0ca-4568-9865-1f7fea1a7149)

- A hint will be to `limit` your result to `1` entry: `LIMIT 1;` so that it returns only the first on the table after correct ordering.